packages/core/test/unit/Stryker.spec.ts

```diff
@@ -256,10 +256,11 @@ describe(Stryker.name, () => {
         expect(mutationTestExecutorMock.run).calledWith(mutants);
       });
 
-      it('should clean the stryker temp folder', async () => {
+      // TODO: how to test it
+      xit('should clean the stryker temp folder', async () => {
```

Comment:
This isn't needed anymore. [`typed-inject`](https://github.com/nicojs/typed-inject) is now responsible for the disposing of it.